tokens/finn.no/datepicker.yml

```diff
@@ -12,4 +12,4 @@ color:
         active: s-color-background-primary
     track:
       background:
-        active: s-color-background-primary-subtle
```

Comment:
Same here regarding moving it to `deprecated.yml`

Reply:
Will reintroduce with todo. 👍